codice/ddf
# Ddf 4158 ui query separation

#### What does this PR do?
Enables UI to work with the new service model where workspaces and queries are separate entities.
#### Who is reviewing it? 
@brianfelix 
@bellcc 
@Corey-Collins 
@glenhein 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining
#### How should this be tested?
1. Create workspace
2. Modify query
3. Save
4. Reload
5. Loaded workspaces and queries should load as expected per modifications from 4.
6. There's no step 6, it's that easy!
#### Any background context you want to provide?
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
